cmd/localInstall.go

```diff
@@ -48,12 +51,20 @@ After that a 'vg ensure' will install like normal again.
 		if len(args) == 2 {
 			path = args[1]
 		} else {
-
-			path = filepath.Join(
-				utils.OriginalGopath(),
-				"src",
-				utils.PkgToDir(pkg),
-			)
+			for _, gopath := range strings.Split(utils.OriginalGopath(), ":") {
```

Comment:
Should use `filepath.SplitList` instead for windows support: https://golang.org/pkg/path/filepath/#SplitList

Reply:
fixed